lms/envs/devstack.py

```diff
@@ -19,7 +19,6 @@
 
 # Don't use S3 in devstack, fall back to filesystem
 del DEFAULT_FILE_STORAGE
-MEDIA_ROOT = "/edx/var/edxapp/uploads"
```

Comment:
File storage has never worked for me on the devstack because of this setting, because the `uploads` directory doesn't exist on the devstack. The default [`MEDIA_ROOT = '/edx/var/edxapp/media/' in `lms.envs.common`](https://github.com/open-craft/edx-platform/blob/jill/TNL-8746/lms/envs/common.py#L1750) does work, however.

I've noticed this issue for a while, but haven't had a chance to address it until now, and since this OSPR relies heavily on file storage, I seized my opportunity.

Reply:
Oh great, it's something that has always caused problems for me too.